cranelift/filetests/src/test_binemit.rs

```diff
@@ -114,6 +115,7 @@ impl binemit::CodeSink for TextSink {
         _: &dyn isa::TargetIsa,
     ) {
     }
+    fn add_call_site(&mut self, _: ir::Opcode, _: ir::SourceLoc) {}
```

Comment:
If we keep this API change, perhaps we could add a default implementation in the trait definition for backward compatibility (I imagine there are other `CodeSink` impls out there among our users)?

Reply:
It's a good idea! Will copy your comment into the upstream PR.